### DISCUSSION TITLE
Splunk url path seems to be hard coded.

### DISCUSSION THREAD

**Discussion by pwills-starz** [moby/moby#45449] [Q&A]
On line 617 it is set to splunkURL.Path = /services/collector/event/1.0 
I am setting up an ECS in AWS and per AWS instructions it says to use the Splunk driver and references this code. 
I need to be able to send the logs to this path splunkURL.Path = /services/collector/raw in order to do line breaking.
There are other options that can be changed but not the path.
Was this done on purpose that the path is not an option to change? Can it be changed?

**Comment by thaJeztah** [comment]
Good question; it looks like it's explicitly disallowed to pass a path at least; https://github.com/moby/moby/blob/1c2d8c8663e63fe8ac19c56fc8e6f48915598a8a/daemon/logger/splunk/splunk.go#L609-L617

- That code was contributed in https://github.com/moby/moby/pull/16488, which ISTR was done by engineers from Splunk, but the event collector was (from that PR) fairly new at the time, so perhaps non-configurable? https://github.com/moby/moby/pull/16488#issuecomment-144516344
